docs/guides/monitoring.md

```diff
@@ -20,7 +22,7 @@ Platformatic can be configured to expose Prometheus metrics:
 ```
 
 In this case, we are exposing the metrics on port 9091 (defaults to `9090`), and we are using basic authentication to protect the endpoint.
-We can also specify the IP address to bind to (defaults to `0.0.0.0`). 
+We can also specify the IP address to bind to (defaults to `0.0.0.0`). We can use `"server": "parent"` to run the `/metrics` endpoint on the same server as platformatic.
```

Comment:
The documentation styleguide says to prefer “You” over “We” but the rest of this section uses “We” everywhere.

Reply:
This page is ok, could you open a separate issue to fix this page?

Reply:
If the page is OK, why open an issue to fix it?

Reply:
Because this PR is not about changing the style of this page, unless you want to take care of it.